src/layers/models/layersManager.ts

```diff
@@ -48,9 +49,8 @@ class LayersManager {
         throw new ConflictError(`Layer name '${layerRequest.name}' is already exists`);
       }
 
-      const newCache: IMapProxyCache = this.getCacheValues(layerRequest.cacheType, layerRequest.tilesPath, layerRequest.format);
+      const newCache: IMapProxyCache = this.getCacheValues(layerRequest.cacheType, layerRequest.tilesPath, this.mapToTileFormat(layerRequest.format));
```

Comment:
separate to a different row "this.mapToTileFormat(layerRequest.format)", not as parameter inside other function

Reply:
done